### PR TITLE
api structuration

### PR DESCRIPTION
**Pull request by Abdur-rahmaanJ**

### CODE REVIEW

**Comment by Abdur-rahmaanJ** [comment]
Few twerks still coming ...

File: shopyo/shopyoapi/utils.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        print("done copying {} to {}".format(source, dest))
+    except Exception as e:
+        print(e)
+
+
+def trycopy(source, dest):
+    try:
+        shutil.copy(source, dest)
+        print("done copying {} to {}".format(source, dest))
+    except Exception as e:
```
Comment (arthurarty) [inline shopyo/shopyoapi/utils.py:16]: @Abdur-rahmaanJ Hey is there a particular exception you are expecting here?

File: shopyo/manage.py [hunk 2]
```diff
@@ -1,136 +1,49 @@
...
-
-
-#@manager.command
+# @manager.command
 def runserver():
     app.run()
 
 
-#@manager.command
+# @manager.command
```
Comment (arthurarty) [inline shopyo/manage.py:24]: @Abdur-rahmaanJ I noticed these had been commented out, but they could be removed completely right?

Comment (Abdur-rahmaanJ) [reply shopyo/manage.py:24]: ah yes yes

File: shopyo/shopyoapi/utils.py (hunk 1 above)
Comment (Abdur-rahmaanJ) [reply shopyo/shopyoapi/utils.py:16]: Thought .copy would throw up errors if file exists but they do not, must maybe remove it

Comment (arthurarty) [reply shopyo/shopyoapi/utils.py:16]: Got it.

**Comment by Abdur-rahmaanJ** [comment]
@arthurarty what do you think of the api-ification?

**Comment by Abdur-rahmaanJ** [comment]
:tada: Travis ok!

**Comment by arthurarty** [comment]
Great.

**Pull request merged**